ls1intum/Artemis
# Hide "Start new assessment" button in Tutor Exercise Dashboard

### Checklist

### Motivation and Context
This PR addresses [this](https://github.com/ls1intum/Artemis/issues/1597) issue.

### Description
1. Hide row to start a new assessment in the Tutor Exercise Dashboard for Team Exercises
2. If the tutor has no assessments yet for a team exercise, show an info box in the location where assessments are started for individual exercises that informs the tutor that assessments are started elsewhere for team exercises (see screenshot below). This should only be seen in very rare cases though since there is now no longer a way to navigate to the Tutor Exercise Dashboard for team exercises via the UI (since the back button no longer redirects the user there).
3. The back button on assessments should lead back to the team page for team exercises (instead of to the Tutor Exercise Dashboard)
4. Perform server-side check if starting assessment is allowed for team exercise also if no result exists yet
5. The "Start new assessment" button on the team page should not be disabled even when the assessment due date has passed. It must be possible for tutors to assess submissions even when they missed the deadline since the teams need to receive an assessment (this is also allowed server-side). This issue was reported by @julian-christl.

### Steps for Testing
1. Log in to Artemis
2. Navigate to Course Management
3. Click on Tutor Course Dashboard (for a course with team exercises)
4. Click on the button "Exercise Teams" for a team exercise
5. Click on the short name of one of the teams
6. Make sure that team has a submission for that exercise
7. Click on the yellow button "Start Assessment" (or "Continue Assessment")
8. Click on the back arrow at the top left and make sure that it redirects back to the team page (and NOT to the tutor exercise dashboard)

### Screenshots
![OnPaste 20200609-182651](https://user-images.githubusercontent.com/7109225/84176955-785c3280-aa82-11ea-9c9d-9c8c20e47c55.png)
